smartmin/users/models.py

```diff
@@ -25,7 +25,7 @@ class RecoveryToken(models.Model):
 
 
 class FailedLogin(models.Model):
-    user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.PROTECT)
+    username = models.CharField(max_length=256)
```

Comment:
can user enter something longer than 256 on login form?

Reply:
Currently django username are up to 150

Reply:
but is that enforced on the form?

Reply:
Yes https://github.com/django/django/blob/fc0c8cfa492b2d7b82263e71e486f3829ad7c43a/django/contrib/auth/forms.py#L196